listen/property-changes.js

```diff
@@ -70,8 +68,9 @@ PropertyChanges.prototype.getOwnPropertyChangeDescriptor = function (key) {
         });
     }
     var objectPropertyChangeDescriptors = this.__propertyChangeListeners__;
-    if (!object_owns.call(objectPropertyChangeDescriptors, key)) {
+    if (!objectPropertyChangeDescriptors.hasOwnProperty(key)) {
```

Comment:
I see... But unless the object was initialized as Object.create(null), objectPropertyChangeDescriptors has the method "hasOwnProperty", so it's kind of arcane...